### DISCUSSION TITLE
Support for react-native-fast-image on Image components

### DISCUSSION THREAD

**Discussion by MuhammedKpln** [wix/react-native-ui-lib#1760] [Ideas]
Is it possible to get react-native-fast-image native module for image components?

**Comment by ethanshar** [comment]
It's definitely something we should consider. 
Maybe as an opt-in feature.

**Reply by MuhammedKpln** [reply]
It will be great  as an opt-in  feature, for people who do not want to install 3rd party native module. But will it be flexibel as it is right now?

**Reply by ethanshar** [reply]
This is something we need to experiment with. 
I created a task for us to explore this option.

**Reply by MuhammedKpln** [reply]
> This is something we need to experiment with. 
> 
> I created a task for us to explore this option.

Glad to hear that! We'll gladly colobrate if needed!
